stdlib/ballerina-database/src/main/ballerina/sql/actions.bal

```diff
@@ -57,7 +57,7 @@ public type CallerActions object {
         R{{}} An `int[]` array of updated row count by each of statements in batch and
                 else `error` will be returned if there is any error
     }
-    public native function batchUpdate(@sensitive string sqlQuery, Parameter[]... parameters) returns int[]|error;
+    public native function batchUpdate(@sensitive string sqlQuery, Param[]... parameters) returns int[]|error;
```

Comment:
Here do we need a change in the API doc?

Reply:
Union type parameters cannot include in documentation due to issue in docerina. So for the moment this is not included.